modules/iap/iap.tf

```diff
@@ -108,6 +108,22 @@ resource "helm_release" "iap" {
   }
 }
 
+resource "kubernetes_annotations" "annotation" {
```

Comment:
Did we resolve the issue with annotations + reapply?

Reply:
No, Umesh is working on it